README.md

```diff
@@ -0,0 +1,30 @@
+# Fury API Blueprint Parser
+
+[![Build Status](http://img.shields.io/travis/refractproject/fury-adapter-apib-parser.svg)](https://travis-ci.org/refractproject/fury-adapter-apib-parser) [![Coverage Status](http://img.shields.io/coveralls/refractproject/fury-adapter-apib-parser.svg)](https://coveralls.io/r/refractproject/fury-adapter-apib-parser) [![NPM version](http://img.shields.io/npm/v/fury-adapter-apib-parser.svg)](https://www.npmjs.org/package/fury-adapter-apib-parser) [![License](http://img.shields.io/npm/l/fury-adapter-apib-parser.svg)](https://www.npmjs.org/package/fury-adapter-apib-parser)
```

Comment:
Super nit, but could you make these `http://` URLs use https.

![screen shot 2015-09-11 at 02 14 58](https://cloud.githubusercontent.com/assets/44164/9811246/ec258504-582a-11e5-95d7-89156d2b7aa4.png)